wradvis/glcanvas.py

```diff
@@ -99,6 +99,7 @@ def __init__(self, **kwargs):
                            method='subdivide',
                            #interpolation='bicubic',
                            cmap=cmap,
+                           clim=(0,10),
```

Comment:
clim=(0, 10) ?

Then I get an almost black image. I'm reading RX in my case. Maybe we should agree on some source dataset? What do you suggest?